public/js/chart-device-data.js

```diff
@@ -137,19 +138,19 @@ $(document).ready(() => {
       console.log(messageData);
 
       // time and temperature are required
-      if (!messageData.MessageDate || !messageData.IotData.ambient.temperature) {
```

Comment:
When testing the sample out, I found it was not nested under "ambient" any more. /shrug